doc/modules/ROOT/pages/python-runtime/knowledge-graph-embeddings.adoc

```diff
@@ -0,0 +1,296 @@
+| int
+| N/A
+| Number of epochs for training (must be greater than 0)
+
+| embedding_dimension
+| int
+| N/A
+| Dimensionality of the embeddings (must be greater than 0)
+
+| epochs_per_checkpoint
+| int
+| max(num_epochs / 10, 1)
+| Number of epochs between checkpoints (must be greater than or equal to 0)
+
+| load_from_checkpoint
+| Optional[tuple[str, int]]
+| None
+| Checkpoint to load from, specified as a tuple (path, epoch)
+
+| split_ratios
+| dict[str, float]
+| {TRAIN=0.8, TEST=0.2}
+| Ratios for splitting the dataset into training and test sets.
+When sum of values is less than 1, the remaining data is used for validation.
+Validation size can be set explicitly with the key `VALID`.
+When all three keys are present, the sum of values must be equal to 1.
+
+| scoring_function
+| str
+| "transe"
+| Function used to score embeddings
+
+| p_norm
+| float
+| 1.0
+| Norm to use in TransE scoring function
+
+| batch_size
+| int
+| 512
+| Size of the training batch (must be greater than 0)
+
+| test_batch_size
+| int
+| 512
+| Size of the test batch (must be greater than 0)
+
+| optimizer
+| str
+| "adam"
+| Optimizer to use for training
+
+| optimizer_kwargs
+| dict[str, Any]
+| {lr=0.01, weight_decay=0.0005}
+| Arguments for the optimizer
+
+| lr_scheduler
+| str
+| ConstantLR
+| Learning rate scheduler
+
+| lr_scheduler_kwargs
+| dict[str, Any]
+| {factor=1, total_iters=1000}
+| Additional arguments for the learning rate scheduler
+
+| loss_function
+| str
+| MarginRanking
+| Loss function to use for training
+
+| loss_function_kwargs
+| dict[str, Any]
+| {margin=1.0, adversarial_temperature=1.0, gamma=20.0}
+| Additional arguments for the loss function
+
+| negative_sampling_size
+| int
+| 1
+| Number of negative samples per positive sample
+
+| use_node_type_aware_sampler
+| bool
+| False
+| Whether to sample negative nodes with the same label as the corresponding positive node
+
+| k_value
+| int
+| 10
+| Value of k used in evaluation metrics
+
+| do_validation
+| bool
+| True
+| Whether to perform validation
+
+| do_test
+| bool
+| True
+| Whether to perform testing
+
+| filtered_metrics
+| bool
+| False
+| Whether to use filtered metrics during evaluation, see <<_filtered_metrics, filtered metrics>>
+
+| epochs_per_val
+| int
+| 50
+| Number of epochs between validations (must be greater than or equal to 0)
+
+| inner_norm
+| bool
+| True
+| Whether to apply normalization to embeddings, see <<_inner_normalisation, inner normalization>>
+
+| init_bound
+| Optional[float]
+| None
+| The value for the range [-bound; bound] of the uniform distribution used to initialize the embeddings. Xavier's initialization is used if None.
+|====
+
+
+.Results
+[opts="header",cols="2m,1,6"]
+|===
+| Name              | Type    | Description
+| modelInfo         | Map     | Details of the trained model.
+| configuration     | Map     | The configuration used to run the procedure.
+| trainMillis       | Integer | Milliseconds to train the model.
```

Comment:
I don't think this is exactly what we have atm? We have some `metadata` field atm I think. Also, Should we still use the Cypher data types when this is just going to be all Python?

Reply:
I removed this chapter, until we have the right one